woocommerce-gateway-stripe.php

```diff
@@ -418,6 +418,49 @@ public function woocommerce_payment_token_set_default( $token_id ) {
 			$stripe_customer->set_default_card( $token->get_token() );
 		}
 	}
+	
+	/**
+	 * Checks Stripe minimum order value authorized per currency
+	 */
+	public function get_minimum_amount() {
```

Comment:
You're going to get warnings because your did not explicitly declare this as static.